app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,7 @@
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"
         tools:ignore="ScopedStorage" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
I thought that `WRITE_EXTERNAL_STORAGE` should already include `READ_EXTERNAL_STORAGE`. I would rather remove the wallpaper code than adding a new permission for it. It's only supported on Android <8.1 anyway.

Reply:
Are you sure? Some people with older devices might not be too happy about it 😅

Reply:
It's just used on the widget settings screen - not really necessary to use the actual background there.

Reply:
Mm, gotcha.